docs/sql/cli.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: SQL CLI
+parent: SQL
+nav_order: 20
+---
+
+# SQL CLI
+
+SQL CLI is a stand-alone Python application that you can launch with the `odfesql` command.
+
+Install the SQL plugin to your Elasticsearch instance, run the CLI using MacOS or Linux, and connect to any valid Elasticsearch end-point.
+
+![SQL CLI](../../images/cli.gif)
+
+## Features
+
+SQL CLI has the following features:
+
+- Multi-line input
+- Autocomplete for SQL syntax and index names
+- Syntax highlighting
+- Formatted output:
+  - Tabular format
+  - Field names with color
+  - Enabled horizontal display (by default) and vertical display when output is too wide for your terminal, for better visualization
+  - Pagination for large output
+- Works with or without security enabled
+- Supports loading configuration files
+- Supports all SQL plugin queries
+
+## Install
+
+To install the CLI:
+
+1. We suggest you install and activate a python3 virtual environment to avoid changing your local environment:
+
+```
+pip install virtualenv
+virtualenv venv
+source ./bin/activate
+```
+
+2. To install the CLI:
+
+```
+pip install odfesql
+```
+
+The SQL CLI only works with Python 3.
+{: .note }
+
+## Using the CLI
+
+1. Index the sample [accounts test data](https://github.com/opendistro-for-elasticsearch/sql/blob/master/src/test/resources/doctest/testdata/accounts.json).
+1. To launch the CLI, run:
+```
+odfesql
+```
+1. Run a sample SQL command:
+```
+SELECT * FROM accounts
```

Comment:
I suggest adding semicolon to the end of the query. The query is only got executed when there is an semicolon and the user hits `enter`. Otherwise, hitting `enter` lead to a new line, which is the feature of "multi-line" imput

Reply:
Done.